examples/hybrid/plane/test_isothermal_channel.jl

```diff
@@ -0,0 +1,353 @@
+
```

Comment:
this file isn't being run as part of the tests.

Reply:
Removed. Will push as separate pr